lib/cli_script.dart

```diff
@@ -398,23 +397,25 @@ IOSink append(String path) => File(path).openWrite(mode: FileMode.append);
 ///
 /// If [name] is passed, it's used as the name of the spawned script.
 ///
+/// [Script.kill] returns `true` if the stream was interrupted and the script
+/// exits with [Script.exitCode] `143`, or `false` if the stream was already
+/// closed.
```

Comment:
In these documentation blocks,I'd lead with the primary effect: "[Script.kill] closes the input stream. It returns ..."